codegen/templates/BulkListExtensions.cs.hbs

```diff
@@ -0,0 +1,31 @@
+// This file is generated automatically, do not edit directly. See codegen/templates/BulkListExtensions.cs.hbs
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OrderCloud.SDK;
+
+namespace OrderCloud.Catalyst
+{
+	public static class ListAllExtensions
+	{
+		private const int MAX_PAGE_SIZE = 100;
+        {{#resources}}{{#with this as | resource |}}{{#operations}}{{#with this as | operation |}}{{#if isList}}       
```

Comment:
What does `#with this as | resource` do? tried googling but it was a bit too generic

Reply:
Ah nvmd, do you don't have to drill into this.resource right? aliasing it to resource

Reply:
yep, aliasing, exactly